examples/TorchDynamo/pippy_dynamo.py

```diff
@@ -20,7 +20,10 @@
 )
 from pippy.microbatch import TensorChunkSpec
 
-import torchdynamo
+import torchdynamo as dynamo
+# TorchDynamo is moved into PyTorch for PyTorch > 1.13
```

Comment:
Curious if there's any constraints for PiPPy to update the pinned version of pytorch and use the latest nightly?

Reply:
I did not update the pinned version of PyTorch in this case because:
Dynamo is not a dependency of PiPPy (at least not right now).
It is just that this example uses Dynamo and PiPPy at the same time.

So, we update the required PyTorch version only when a module PiPPy depends on need an upgrade.